### PR TITLE
added dynamic cmd

### PR DESCRIPTION
**Pull request by ordinall**

### CODE REVIEW

File: lib/rtsp-ffmpeg.js [hunk 1]
```diff
@@ -39,12 +40,6 @@ var FFMpeg = function(options) {
 
 util.inherits(FFMpeg, EventEmitter);
 
-/**
- * FFMpeg command name
- * @type {string}
- */
-FFMpeg.cmd = 'ffmpeg';
```
Comment (agsh) [inline lib/rtsp-ffmpeg.js]: I'd preffered to remain this to support backward compatibility and add above something like this
`this.cmd = options.cmd || FFMpeg.cmd || 'ffmpeg';`

**Review by agsh — COMMENTED** [review]
Thank you! Great PR! Why not, never thought about this :smile: I only have one suggestion in the review

**Comment by ordinall** [comment]
i had a use case where i had to use ffmpeg-static's given path as ffmpeg binary instead of default having ffmpeg in path. Hence, this. One improvement over this is maybe have something with env variable too, by default its 'ffmpeg', if environment variable 'FFMPEG_PATH' is set use that. if given a cmd as parameter prioritise that.

**Comment by agsh** [comment]
Thanks!

**Pull request merged**